tests/cpgrid/grid_lgr_test.cpp

```diff
@@ -592,9 +612,7 @@ BOOST_AUTO_TEST_CASE(pathces_share_face)
     const std::vector<std::array<int,3>> startIJK_vec = {{0,0,0}, {2,0,0}, {3,2,2}};
     const std::vector<std::array<int,3>> endIJK_vec = {{2,1,1}, {3,1,1}, {4,3,3}};
     const std::vector<std::string> lgr_name_vec = {"LGR1", "LGR2", "LGR3"};
-    BOOST_CHECK_THROW(coarse_grid.addLgrsUpdateLeafView(cells_per_dim_vec, startIJK_vec, endIJK_vec, lgr_name_vec), std::logic_error);
-    BOOST_CHECK_EQUAL(coarse_grid.chooseData()[0]->patchesShareFace(startIJK_vec, endIJK_vec), true);
```

Comment:
They are gone since now we do support blocks of cells to be refined sharing faces on their boundaries :)
![Screenshot from 2024-06-07 16-47-12](https://github.com/OPM/opm-grid/assets/89510440/efbe1479-af69-49a5-aec9-0439ccf166b8)